common/centraldashboard/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- base/serviceaccount.yaml
+- base/role.yaml
+- base/deployment.yaml
+- base/rolebinding.yaml
+- base/service.yaml
+commonLabels:
+  kustomize.component: centraldashboard
+images:
+  - name: gcr.io/kubeflow-images-public/centraldashboard
+    newName: gcr.io/kubeflow-images-public/centraldashboard
+    newTag: latest
```

Comment:
Any particular reason you are defaulting to latest here? I think we always to pin to specific images so that the behavior is deterministic.